Jumpscale/clients/zero_hub/README.md

```diff
@@ -8,51 +8,46 @@ pip3 install -e 'git+https://github.com/threefoldtech/0-hub#egg=zerohub&subdirec
 
 ## Using the client
 
+### Creating the client in JSX
+
+```
+zhubcl = j.clients.zhub.zhubcl
+zhubcl.username = "$your_zhub_username"
+zhubcl.token = "$your_itsyouonline_token"
+
+zhubcl.save()
+```
+
 ### Public
-You can make basic requests without authentification like:
+You can make basic requests without authentication like:
 ```
-cl = j.clients.zhub.getClient()
+cl = j.clients.zhub.getClient('$your_zhub_client_name')
 
 cl.repositories()
 cl.list()
 cl.list('maxux')
 cl.get('maxux', 'ubuntu1604.flist')
 ```
 
-### Authentification
-In order to use upload, delete, ... feature, you need to authentificate yourself.
-There are two ways to authentificate youself:
-
-- By providing a token during `getClient`
-- By authentificate yourself on an existing client with: `cl.authentificate()`
+### Authentication
+In order to use upload, delete, ... feature, you need to authenticate yourself by setting a token on your client.
 
-Note: the second method allows to change your username and enable switch-user feature.
-
-Example:
 ```
-cl = j.clients.zerohub.getClient('jwt-token')
-
-cl.upload('/tmp/my-upload-file.tar.gz')
-cl.rename('my-upload-file.flist', 'my-super-flist.flist')
+iyo_client = j.clients.itsyouonline
+token = iyo_client.jwt_get().jwt
```

Comment:
as stated in #188 
>the jwt should also only have the `user:memberof:tf-official-apps` scope when uploading to tf-official apps, not the default`user:admin`scope

Reply:
added in https://github.com/threefoldtech/jumpscaleX/pull/203/commits/576c74ea25cf1ec2ec9e64c82a3d4ef672f19ca9